src/ui/Onboarding/shared/useMnemonicInput.tsx

```diff
@@ -8,11 +8,13 @@ export function useMnemonicInput({
   rows = 4,
   maxInputNumber = 24,
   setValue,
+  showCleanedClipboardMessage,
 }: {
   columns?: number;
   rows?: number;
   maxInputNumber?: number;
   setValue: Dispatch<SetStateAction<string[]>>;
+  showCleanedClipboardMessage?: boolean;
```

Comment:
maybe simply `showClipboardMessage`?

Or another way:
```
type?: 'text' | 'password'
```

So if type is passed, use it, otherwise follow existing logic for `type`.